lib/Serialization/ModuleFormat.h

```diff
@@ -1487,12 +1487,12 @@ namespace decls_block {
     // The sub-pattern trails the record.
   >;
 
-  using VarPatternLayout = BCRecordLayout<
-    VAR_PATTERN,
-    BCFixed<1>, // isLet?
-    BCFixed<1>  // implicit?
-    // The sub-pattern trails the record.
-  >;
+  using VarPatternLayout = BCRecordLayout<VAR_PATTERN,
```

Comment:
Please increment the module format version.

Reply:
In practice the only patterns we serialize are for globals and members of types, which won't have the 'shared' bit set. Perhaps we should assert that it's never set when serializing instead, since it doesn't really make sense to have a non-local borrowed value like that.

Reply:
@slavapestov do you mean I should add an assert and remove the `isShared` bit?

Reply:
Yes, I think that makes sense. There is nothing new to serialize here.